dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -846,14 +847,17 @@ private WorkflowVersion setupWorkflowFilesForGitHubVersion(Triple<String, Date,
             }
             theWf = maybeWorkflow.get();
             testParameterPaths = theWf.getTestParameterFiles();
+            if (testParameterPaths != null) {
+                testParameterPaths = testParameterPaths.stream().map(this::convertToAbsolutePath).collect(Collectors.toList());
+            }
         } catch (DockstoreYamlHelper.DockstoreYamlException ex) {
             String msg = "Invalid .dockstore.yml: " + ex.getMessage();
             LOG.info(msg, ex);
             return null;
         }
 
         // No need to check for null, has been validated
-        String primaryDescriptorPath = theWf.getPrimaryDescriptorPath();
+        String primaryDescriptorPath = convertToAbsolutePath(theWf.getPrimaryDescriptorPath());
```

Comment:
Comment/question for line 866/870 (I can't comment directly there, because it's not close enough to your change): We will now be storing the absolute path in the path field. Do we want that? I don't know, I'm not clear on where the differences between path and absolute path matter.

Also, I'd probably call this variable something like `primaryDescriptorAbsolutePath` -- then it would be clear on line 866/870 that you're setting the path to an absolute path (if that's what we want).